bin/restore_cache

```diff
@@ -4,12 +4,12 @@ CACHE_KEY=$1
 
 S3_BUCKET_NAME=${CACHE_BUCKET_NAME-}
 if [ -z "$S3_BUCKET_NAME" ]; then
-	if [ -z "$BUILDKITE_PLUGIN_BASH_CACHE_BUCKET" ]; then
+	if [ -z "$BUILDKITE_PLUGIN_A8C_CI_TOOLKIT_BUCKET" ]; then
 		echo "⛔Unable to restore file from cache – no \$CACHE_BUCKET_NAME is set"
 		exit 1
 	else
-		echo "Reading bucket name from 'BUILDKITE_PLUGIN_BASH_CACHE_BUCKET'"
-		S3_BUCKET_NAME="$BUILDKITE_PLUGIN_BASH_CACHE_BUCKET"
+		echo "Reading bucket name from 'BUILDKITE_PLUGIN_A8C_CI_TOOLKIT_BUCKET'"
+		S3_BUCKET_NAME="$BUILDKITE_PLUGIN_A8C_CI_TOOLKIT_BUCKET"
```

Comment:
Unless I'm mistaken, this code uses `$BUILDKITE_PLUGIN_A8C_CI_TOOLKIT_BUCKET` only if `$CACHE_BUCKET_NAME` is empty (`-z`).